chapter1/web-search-agent/examples.py

```
@@ -33,10 +33,14 @@ def batch_search(self, questions: List[str]) -> List[Dict[str, str]]:
             logger.info(f"处理问题 {i}/{len(questions)}: {question}")
             try:
                 answer = self.search_and_answer(question)
+                # search_and_answer 内部已捕获异常并返回错误字符串（见 agent.py），
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)